src/constants/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+export const DEFAULT_ASSETS_NUMBER_PER_PAGE = 50;
+export const FILE_CHUNK_SIZE = 1024 * 1024 * 5;
+export const WEBSOCKET_TOPIC_NAME = 'files-messages';
```

Comment:
Probably a leftover from when we wanted to do the websockets approach?

Reply:
Removed.